docs/api/browser-window-proxy.md

```diff
@@ -50,4 +50,4 @@ The `BrowserWindowProxy` object has the following instance properties:
 
 #### `win.closed`
 
-A Boolean that is set to true after the child window gets closed.
+A `Boolean` that is set to true after the child window gets closed.
```

Comment:
@zeke 

What I have that the structured docs don't provide is
* optional
* array yes/no (I can parse that out of the type)
* newlines, lists and programlistings in the description, so I can properly format the paragraphs like seen [here](https://github.com/fab1an/kotlin-electron-api/blob/master/generated-api/jsapi/electron/browser-window.kt#L593)

What I don't have (yet) is
* information about possible values for a String (enums)
* platform information

Some other problems: [Here](https://github.com/electron/electron-api-docs/blob/master/electron-api.json#L2730) the `array` information isn't extracted out of the union type. (Also the api isn't clear in whether an array of only one of the types or a mixed-type array can be passed)

Both parsers seem pretty complete though, we can probably generate the structured docs out of my files or the other way round.